proto/README.md

```diff
@@ -0,0 +1,14 @@
+# Code generation
+
+Protobuf code is generated locally using Buf. There are two reasons why 
+we chose this over Buf's remote packages:
```

Comment:
I don't think we need to justify our decision why we use the approach that we do. I'd rather just explain how to generate the code (although that's pretty simple and already explained in the main readme, so maybe even that would be redundant).

Reply:
The reason why I've put this is that we may in future want to revisit this decision, so knowing why we made this decision at this point would provide useful insight.

Reply:
I'd argue that's what issues and PRs are for. And given the conversations we had with the buf team it seems like our current approach is "correct", it's expected to generate the code locally and use remote generation only in external projects, so we are kinda explaining why we are doing things the right way. If we were doing something out of the ordinary I wouldn't complain, at least not so loudly 😄

Reply:
I'd say that such decisions are easier to find in a doc file in the repo itself than by searching. But I don't feel too strongly about that and it's not a big one anyway.:) Oh, and related to what the Buf team said: they also suggested using draft commits (which are now GA), but that doesn't quite work either.